File: sdsServer/src/main/java/com/sds/amasoft/dto/SolicitationRequest.java
package com.sds.amasoft.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDate;
import java.time.LocalTime;

@NoArgsConstructor
@Getter
@Setter
public class SolicitationRequest {
    //    private static final long serialVersionUID = 65981149772133526L;
    private Long serviceId;

    private Long userId;
    private LocalDate date;
    private LocalTime time;
}

File: sdsWeb/src/app/model/SolicitationRequest.ts
export class SolicitationRequest {
  serviceId: any;
  userId: any;
  date: any;
  time: any;

  constructor(obj?: any) {
    this.serviceId = obj?.serviceId || null;
    this.userId = obj?.userId || null;
    this.date = obj?.date || null;
    this.time = obj?.time || null;
  }
}
